app/scripts/models/auth_brokers/fx-sync-channel.js

```diff
@@ -131,6 +134,30 @@ define(function (require, exports, module) {
         .then(() => proto.beforeSignUpConfirmationPoll.call(this, account));
     },
 
+    afterSignInConfirmationPoll (account) {
+      return proto.afterSignInConfirmationPoll.call(this, account)
+        .then((behavior) => {
+          if (this.hasCapability('sendAfterSignInConfirmationPollNotice')) {
+            const loginData = this._getLoginData(account);
+            return this.send(this.getCommand('EMAIL_VERIFIED'), loginData)
```

Comment:
Pulling [over the comment from the issue](https://github.com/mozilla/fxa-content-server/issues/5029#issuecomment-308397925):

@mcomella - if we changed the WebChannel message from fxaccounts:email_verified to fxaccounts:verified, would that cause havoc for you?